_studio/mfx_lib/decode/mpeg2/src/mfx_mpeg2_decode.cpp

```diff
@@ -1030,9 +1043,10 @@ void VideoDECODEMPEG2::FillOutputSurface(mfxFrameSurface1* surface_work, mfxFram
 {
     const auto fd = frame->GetFrameData();
```

Comment:
at his point you used ```frame``` which can be null (because it is input you can't rely on outside code). As alternative you can use reference here